doc/radosgw/rgw-cache.rst

```diff
@@ -6,10 +6,10 @@ RGW Data caching and CDN
 
 .. contents::
 
-This new feature adds to RGW the ability to securely cache objects and offload the workload from the cluster, using Nginx.
-After an object is accessed the first time it will be stored on top of Nginx dir.
-Every request the Nginx will not fetch data from the RGW or the cluster, It will only check for permissions for that user from the RGW.
-This feature is based on some Nginx modules, ngx_http_auth_request_module, https://github.com/kaltura/nginx-aws-auth-module, Openresty for lua capablities.     
+This feature adds to RGW the ability to securely cache objects and offload the workload from the cluster, using Nginx.
+After an object is accessed the first time it will be stored in Nginx directory.
```

Comment:
s/in Nginx directory/in the Nginx directory;  (taking "directory" to be the abstraction, not one POSIX directory)